src/github.com/getlantern/errlog/errlog.go

```diff
@@ -0,0 +1,529 @@
+/*
+Package errlog defines error types used across Lantern project and implements
+functions to manipulate them.
+
+  var elog = errlog.ErrorLoggerFor("package-name")
+  if n, err := Foo(); err != nil {
+    elog.Log(err, errlog.WithOp("foo"), errlog.WithField("foo": "bar"))
+  }
+
```

Comment:
Because in most cases simply `elog.Log(err)` would be enough. Different set of extra parameters can be supplied in different cases. For example, user agent is useful only when error interacting with client connections.